core/invoker/src/main/scala/whisk/core/invoker/InvokerReactive.scala

```diff
@@ -72,7 +72,7 @@ class InvokerReactive(
 
     /** Cleans up all running wsk_ containers */
     def cleanup() = {
-        val cleaning = docker.ps(Seq("name" -> "wsk_"))(TransactionId.invokerNanny).flatMap { containers =>
+        val cleaning = docker.ps(Seq("name" -> s"wsk${instance.toInt.toString()}_"))(TransactionId.invokerNanny).flatMap { containers =>
```

Comment:
Please omit the `toString()`, it's not needed inside this string interpolation, the integer is fine.